lib/run/summary.js

```diff
@@ -267,13 +267,28 @@ _.assign(RunSummary, {
     attachTimingTrackers (summary, emitter) {
         // mark the point when the run started
         // also mark the point when run completed and also store error if needed
-        emitter.on('start', function () { summary.run.timings.started = Date.now(); });
+        var x = 0;
+
+        emitter.on('start', function () {
+            summary.run.timings.started = Date.now();
+            summary.run.timings.iterationData = [];
```

Comment:
Call this variable "iterations" please. We are storing "summary" of "run" "timings" for "iterations". 😊